cmd/nerdctl/run.go

```diff
@@ -758,23 +774,14 @@ func withBindMountHostProcfs(_ context.Context, _ oci.Client, _ *containers.Cont
 	return nil
 }
 
-func generateLogURI(dataStore, logDriver string, logOptMap map[string]string) (*url.URL, error) {
-	var selfExe string
-	if logDriver == "json-file" {
-		var err error
-		selfExe, err = os.Executable()
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		return nil, fmt.Errorf("%s is not yet supported", logDriver)
+func generateLogURI(dataStore string) (*url.URL, error) {
+	selfExe, err := os.Executable()
+	if err != nil {
+		return nil, err
 	}
 	args := map[string]string{
 		logging.MagicArgv1: dataStore,
```

Comment:
Why not consider splicing the parameters behind the dataStore, like
`binary:///usr/local/bin/nerdctl?_NERDCTL_INTERNAL_LOGGING=%2Fvar%2Flib%2Fnerdctl%2F1935db59%3FmaxSize%3D1
`

Reply:
Just didn't want to invent a new microformat

Reply:
Or put args in the labels of the container?

To be honest, I am not inclined to store some information by adding new files. :(

Reply:
How can we let the log binary access the labels?

Reply:
It's a bit tricky, ignore it